x-pack/plugins/monitoring/server/lib/kibana/get_kibanas_for_clusters.ts

```diff
@@ -25,7 +25,7 @@ import { Globals } from '../../static_globals';
  *  - number of instances
  *  - combined health
  */
-export function getKibanasForClusters(req: LegacyRequest, clusters: Cluster[], ccs: string) {
+export function getKibanasForClusters(req: LegacyRequest, clusters: Cluster[], ccs?: string) {
```

Comment:
`ccs` is already optional within the functions that this function calls. `getKibanasForClusters` is called in one other location where `ccs` is supplied though, so rather than remove the parameter, I've just made it optional.